docker-app/qfieldcloud/core/tests/test_delta.py

```diff
@@ -590,13 +589,13 @@ def test_change_and_delete_pushed_only_features(self):
     def get_file_contents(self, project, filename):
         response = self.client.get(f"/api/v1/files/{project.id}/{filename}/")
 
-        if isinstance(response, HttpResponseRedirect):
```

Comment:
For next time, it's cleaner to set `self.client.get(f"/api/v1/files/{project.id}/{filename}/", follow=True)`